apps/storefront/app/grunnleggende/introduksjon/om-designsystemet/page.mdx

```diff
@@ -118,13 +118,15 @@ Vi vil gjerne høre fra deg, [send oss en epost](mailto:designsystem@digdir.no)!
 
 ## Hvordan samarbeider vi på tvers?
 
-En viktig verdi for oss er å være åpne og transparente, og vi tror dette er avgjørende for å få til et samarbeid på tvers. Alt vi gjør deler vi åpent - [kildekode](https://github.com/digdir/designsystemet), [backlog](https://github.com/orgs/digdir/projects/3/views/1), [roadmap](https://github.com/digdir/designsystemet/issues/1435), [Figma](https://www.figma.com/@designsystemet), [Storybook](https://storybook.designsystemet.no).
+En viktig verdi for oss er å være åpne og transparente. Alt vi gjør deler vi åpent - [kildekode](https://github.com/digdir/designsystemet), [backlog](https://github.com/orgs/digdir/projects/3/views/1), [roadmap](https://github.com/orgs/digdir/projects/3/views/30), [Figma](https://www.figma.com/@designsystemet), [Storybook](https://storybook.designsystemet.no) og [diskusjoner i slack](https://www.designsystemet.no/slack). 
```

Comment:
We also have https://github.com/digdir/designsystemet/discussions ?

Maybe this paragraph can be a list or something in the future

Reply:
Eg støtter liste på dette her